docs/content/en/connecting_your_tools/parsers/file/anchore_engine.md

```diff
@@ -6,35 +6,50 @@ toc_hide: true
 ### File Types
 DefectDojo parser accepts a .json file.
 
-Using the [Anchore CLI](https://docs.anchore.com/current/docs/using/cli_usage/images/inspecting_image_content/) is the most reliable way to generate an Anchore report which DefectDojo can parse. When generating a report with the Anchore CLI, please use the following command to ensure complete data: `anchore-cli --json image vuln <image:tag> all`
```

Comment:
Could you please update your PR to fit both versions of the Anchore parser: the old Anchore CLI and the new Achnore UI.
Also, please add unittests to the PR to fit both versions.

Reply:
I agree with @manuel-sommer here - we need to support both formats since we can't know which on a DefectDojo is using.